ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -508,16 +508,11 @@ postldap__ldap_start_tls: '{{ ansible_local.ldap.start_tls
 # .. envvar:: postldap__ldap_user_filter [[[
 #
 # The LDAP filter used to look up user accounts in the directory.
-postldap__ldap_user_filter: '(&
-                                           (objectClass=inetOrgPerson)
-                                           (objectClass=postfixUser)
-                                         )
-                                         (mailEnabled=TRUE)
-                                         (|
-                                           (authorizedService=postfix)
-                                           (authorizedService=mail)
-                                           (authorizedService=\*)
-                                         )'
+postldap__ldap_user_filter: '(|
+                               (authorizedService=postfix)
+                               (authorizedService=mail)
+                               (authorizedService=\*)
+                             )'
```

Comment:
I think that now the user filter is too broad and would let almost any LDAP object be treated as a mail account. Check the filter [from the `debops.saslauthd` role](https://github.com/debops/debops/blob/master/ansible/roles/debops.saslauthd/defaults/main.yml#L294-L313):
```yaml
      - name: 'ldap_filter'
        value: '(|
                  (&
                    (objectClass=inetOrgPerson)
                    (uid=%U)
                    (|
                      (authorizedService=smtpd)
                      (authorizedService=\2a)
                    )
                  )
                  (&
                    (objectClass=account)
                    (uid=%U)
                    (host=%r)
                    (|
                      (authorizedService=smtpd)
                      (authorizedService=\2a)
                    )
                  )
                )'
```
This filter is very narrow, but supports both People and Service accounts, with different attributes. But this filter is focused on mail submission authentication, and i don't think this would be a good fit for mail accounts because services that send mail don't necessarily have to have real mailboxes.

I would probably replace the `(objectClass=account)` part with something focused on functional accounts, think "marketing", "finances", "support", etc. Basicaly department-based accounts which might be shared between a few users. I think that a good way to support them would be to include support for [shared mailboxes](https://vogasec.wordpress.com/2012/07/01/ubuntu-postfix-dovecot-shared-mailboxes/) in Dovecot. But that's out of scope of this PR.

Also, changing the `authorizedService=postfix` to `authorizedService=smtpd` would make the attribute values more agnostic; other SMTP servers could support the same filter.

Reply:
It is a chicken egg problem.... Will change as proposed

Reply:
This is not a chicken-egg problem, you don't initialize mail accounts in LDAP via Postfix. Besides, DebOps being in a monorepo lets us create a single definition of an e-mail account and keep it coherent across multiple Ansible roles. Nothing chikeny about that, I'm afraid. :-)

Reply:
I was actually referring to the "Problem" of the unfinished LDAP schema.